src/app/manager-page/student-card/student-card.component.html

```diff
@@ -0,0 +1,19 @@
+<div *ngIf="this.student" class="mt-5 shadow bg-white rounded">
+  <div class="d-flex justify-content-between">
+    <img class="m-4"src="{{student.pictureUrl}}" style="border-radius:50%;" width="300" height="300">
```

Comment:
maybe move this styling to ```student-card.component.scss```?
```suggestion
    <img class="m-4"src="{{student.pictureUrl}}">
```